packaging/suse/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: Apache-2.0
+#!BuildTag: trento/trento-wanda:latest
+#!BuildTag: trento/trento-wanda:%%VERSION%%
+#!BuildTag: trento/trento-wanda:%%VERSION%%-build%RELEASE%
+#!UseOBSRepositories
+
+FROM bci/rust:1.66 AS release
```

Comment:
Let's test this first in IBS before merging

Reply:
Tried and seems to be working https://build.opensuse.org/package/show/home:nkopliku:trento-factory/trento-wanda-image

Reply:
That's an OBS url. Not IBS, which has a different repositories/package configuration

Reply:
True :sweat_smile: I sent you the link to IBS